test/testcafe/framework/page-objects/YubiKeyAuthenticatorPageObject.js

```diff
@@ -9,7 +10,22 @@ export default class YubiKeyAuthenticatorPageObject extends BasePageObject {
     return this.form.setTextBoxValue(name, value);
   }
 
-  submit() {
-    return this.form.clickSaveButton();
+  async waitForYubikeyView() {
```

Comment:
Why did you have to add this wait? Doesn't the navigate check console and wait for the form to render?

Reply:
the `checkConsoleMessages` tests were failing unless i specifically waited for the form to render
![image](https://user-images.githubusercontent.com/107433508/201211819-51454b49-eba7-4a5d-ab9f-c1e8df63629c.png)

Reply:
seems like the navigate doesnt wait for the form to fully render

Reply:
I moved the `checkConsoleMessages` method lower to wait for the form to fully load and eliminated the need for this `waitForYubikeyView` method